extensions/blocks/publicize/editor.scss

```diff
@@ -30,12 +30,15 @@
 	&.is-facebook {
 		fill: var( --color-facebook );
 	}
+
```

Comment:
Just double checking: I assume these spaces were automagically added by the linter, right?

Reply:
Indeed